frontend/src/metabase/containers/UndoListing.tsx

```diff
@@ -168,8 +168,16 @@ function UndoListingInner() {
           onDismiss={() => dispatch(dismissUndo({ undoId: undo.id }))}
         />
       ))}
-    </UndoList>
+    </FloatingUndoList>
   );
 }
 
-export const UndoListing = BodyComponent(UndoListingInner);
+export const FloatingUndoList = ({ children }: { children: ReactNode }) => {
```

Comment:
do you plan to use this component elsewhere? If not, it shouldn't be exported

```suggestion
const FloatingUndoList = ({ children }: { children: ReactNode }) => {
```

Reply:
Yes, I'll use it in a downstream PR. I try to avoid adding changes that only make sense in light of downstream stuff, but maybe this one is OK

Reply:
("downstream" in the sense of "later, in the stack of PRs")